Frontend/CO.CDP.OrganisationApp/Pages/Supplier/SupplierVat.cshtml.cs

```diff
@@ -0,0 +1,84 @@
+using CO.CDP.Mvc.Validation;
+using CO.CDP.Organisation.WebApiClient;
+using Microsoft.AspNetCore.Mvc;
+using System.ComponentModel.DataAnnotations;
+
+namespace CO.CDP.OrganisationApp.Pages;
```

Comment:
not sure is this namespace correct or it should be something like this 

namespace CO.CDP.OrganisationApp.Pages.Supplier;